### DISCUSSION TITLE
why do we use '' and sometimes not when we are doing variables with numbers

### DISCUSSION THREAD

**Discussion by Helenmoon33** [community/community#86982] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

why do we use '' and sometimes not when we are doing variables with numbers, for example variable is months = 12 print (months) but sometimes I see that its '12' not 12, and sometimes I see ('months') and (months)

**Comment by faridnec** [comment, marked as answer]
In Python (example), the use of quotes around numbers or variable names in certain contexts can have different meanings
- Without quotes: Refers to the variable or numeric value itself.
- With single quotes: Treats the content as a string.
- Quotes around variable name: Treats the variable name as a string.

in the scenario when comparing '12' and 12 using the equality operator (==) would yield False because one is a string and the other is an integer. Understanding the context in which you're using quotes helps you choose the appropriate representation for your data.

**Comment by Helenmoon33** [comment]
ohhhhhh ok cool thank you I understand that
